### ISSUE TITLE
Can't get desired sound for FCM

### ISSUE THREAD

**Issue by AndrewHenderson** [invertase/react-native-firebase#142]
I'm trying to use the default "note" sound for FCM. I can't find any documentation on Firebase or here demonstrating how to access system sounds.

```js
{
  notification: {
    title: notification.title,
    body: notification.body,
    badge: String(_.size(unreadNotifications)),
    sound: 'note'
  }
};
```
I've found this list of system sounds and paths: http://iphonedevwiki.net/index.php/AudioServices

And this [Apple Documentation](https://developer.apple.com/library/content/documentation/NetworkingInternet/Conceptual/RemoteNotificationsPG/SchedulingandHandlingLocalNotifications.html#//apple_ref/doc/uid/TP40008194-CH5-SW3)

I've also posted on SO: https://stackoverflow.com/questions/44211085/where-can-i-find-documentation-on-firebase-cloud-messaging-available-sounds

**Comment by chrisbianca** [comment]
@AndrewHenderson The notification sound is managed completely by the OS, so no way of being influenced by RN-Firebase directly.  The documentation here https://firebase.google.com/docs/cloud-messaging/http-server-ref#notification-payload-support it looks like you're doing the right thing, so it might be an FCM issue itself which would be worth contacting Firebase directly

**Comment by AndrewHenderson** [comment]
@chrisbianca I've contacted Firebase. Here's what they replied:

> FCM doesn't have a list of string values that you could use for the sound of your notification. Sound parameter in FCM could look into the sound files in the main bundle of your app or in the Library/Sounds folder of the app's data container. Please refer to the [iOS Developer Library](https://developer.apple.com/library/content/documentation/NetworkingInternet/Conceptual/RemoteNotificationsPG/SupportingNotificationsinYourApp.html#//apple_ref/doc/uid/TP40008194-CH4-SW10) for more information.

I sent the following and am awaiting a response:

> I'm not looking to have any custom sounds, but rather I'm trying to have FCM look into the platform's available sounds. Is this possible?

**Comment by chrisbianca** [comment]
Ok, well let us know what you find, but I'm going to close this issue as it's not a react-native-firebase issue.

**Comment by AndrewHenderson** [comment]
@chrisbianca I imagine this is a very common desire – to select the sound you want your client-side FCM code to trigger when a user receives a notification.

I don't mean to be a bother, but I really don't know who better to ask than Firebase and this library which I rely upon heavily.

**Comment by chrisbianca** [comment]
I can't see any reference to the "note" sound in the documentation you sent through?  
According to this: https://stackoverflow.com/a/18294871/5909648 you can't use system sounds other than default in your app.  I'd suggest bundling in your own custom sound if you want to do something like that...

**Comment by AndrewHenderson** [comment]
Thanks for this link. Note was just a guess based on the sound name in my iPhone's GUI. I tried `ReceivedMessage` based on the following list: http://iphonedevwiki.net/index.php/AudioServices which did not work either.

I think that it might be possible based on the [comment above](https://stackoverflow.com/a/15642089/933711) the one you linked to and the update in the comment you linked to:

> While Apple does say you can't access system sounds, I suppose they must mean that you can't access them for your own purposes, but I see that UILocalNotification.h provides:

> UIKIT_EXTERN NSString *const UILocalNotificationDefaultSoundName; 
"Identifies the default system sound to play when a notification alert is displayed. You assign this value to the soundName property." I am now using this in my app and it seems to work fine.

Many of the apps I use leverage system sounds. I can't imagine they all copy system sounds into their app. Perhaps there's a limitation to frameworks as someone mentioned.

**Comment by chrisbianca** [comment]
Did you try specifying the actual filename `ReceivedMessage.caf`? Or just used `ReceivedMessage`?  Do note that the list you're referring to talks about sound available in iOS 2.0 - 5.0 beta, so they may be well out of date!

**Comment by cikcoh** [comment]
@AndrewHenderson Hi, did u managed to solve this? i trying to figure out where to put the sound file, for now it only plays the default beep
